package.json

```diff
@@ -10,12 +10,14 @@
   },
   "dependencies": {
     "axios": "^1.7.7",
+    "clsx": "^2.1.1",
     "framer-motion": "^7.3.5",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-icons": "^4.4.0",
     "react-lottie-player": "1.5.4",
-    "smoothscroll-polyfill": "^0.4.4"
+    "smoothscroll-polyfill": "^0.4.4",
+    "tailwind-merge": "^2.5.4"
```

Comment:
I would avoid adding another two libraries for something that is just being used at one place and can be achieved otherwise as well. 

So let's remove these and use the basics itself :)

Reply:
I understood why you used them 

But the same can be achieved without these libraries also. So lets avoid using that.